### PR TITLE
Add flexible load for lr scheduler.

### PR DESCRIPTION
**Pull request by veichta**

### CODE REVIEW

File: gluefactory/train.py [hunk 1]
```diff
@@ -48,11 +48,12 @@
     "optimizer_options": {},  # optional arguments passed to the optimizer
     "lr": 0.001,  # learning rate
     "lr_schedule": {
-        "type": None,
+        "type": None,  # can be any of [factor, exp] or torch.optim.lr_scheduler
         "start": 0,
         "exp_div_10": 0,
         "on_epoch": False,
         "factor": 1.0,
+        "options": None,  # add lr_scheduler arguments here
```
Comment (sarlinpe) [inline gluefactory/train.py:56]: Shouldn't this be `{}` by default?

File: gluefactory/train.py [hunk 2]
```diff
@@ -48,11 +48,12 @@
     "optimizer_options": {},  # optional arguments passed to the optimizer
     "lr": 0.001,  # learning rate
     "lr_schedule": {
-        "type": None,
+        "type": None,  # can be any of [factor, exp] or torch.optim.lr_scheduler
```
Comment (sarlinpe) [inline gluefactory/train.py:51]:
```suggestion
        "type": None,  # string in {factor, exp, member of torch.optim.lr_scheduler}
```

File: gluefactory/train.py (hunk 1 above)
Comment (veichta) [reply gluefactory/train.py:56]: Yes that would be more clean.

```suggestion
        "options": {},  # add lr_scheduler arguments here
```

**Review by sarlinpe — APPROVED** [review]
